docs/02-app/01-building-your-application/07-configuring/01-typescript.mdx

```diff
@@ -319,6 +319,7 @@ When you need to declare custom types, you might be tempted to modify `next-env.
 
 | Version   | Changes                                                                                                                              |
 | --------- | ------------------------------------------------------------------------------------------------------------------------------------ |
+| `v15.0.0` | `next.config.ts` support added for TypeScript projects.                                                                              |
```

Comment:
should we publish this later?

Reply:
This is perfect, thank you!

We can publish now since the changes will only be available in the RC docs: https://rc.nextjs.org/